src/Core/Commander/Interfaces/ApiInterface/ApiGlobe.js

```diff
@@ -169,12 +178,44 @@ ApiGlobe.prototype.getMinZoomLevel = function(id) {
  * @constructor
  * @param {id} id - The id of the layer.
  */
+ApiGlobe.prototype.addTileMatrixSet = function(options){
+    var provider = this.scene.managerCommand.getProtocolProvider("wmts");
+        provider.addTileMatrixSets(options);
+};
+
+ApiGlobe.prototype.getTileMatrixSet = function(id){
+    var provider = this.scene.managerCommand.getProtocolProvider("wmts");
+    return provider.getTileMatrixSets(id);
+};
+
+ApiGlobe.prototype.addFeature = function(options){
+    if(options === undefined)
+        throw new Error("options is required");
+    var map = this.scene.getMap();
+    map.addFeature(options);
+};
 
 ApiGlobe.prototype.getLayers = function( /*param*/ ) {
     var map = this.scene.getMap();
     return map.layersConfiguration.getColorLayers();
 };
 
+
+ApiGlobe.prototype.pickFeature = function(Position, layerId){
+    if(Position == undefined)
+       throw new Error("Position is required");
+    var map = this.scene.getMap();
+    var layer = map.getFeatureLayerByName(layerId);
+    return this.scene.getPickFeature(Position, layer);
+};
+
+ApiGlobe.prototype.removeFeature = function(featureId, layerId){
```

Comment:
removeFeature(feature) have only one parameter.
Feature have layerId and featureId.